jekyll/_cci2/building-docker-images.md

```diff
@@ -111,30 +111,14 @@ Let’s break down what’s happening during this build’s execution:
 
 ## Docker Version
 
-If your job requires a specific docker image, you can set it as a `version` attribute:
```

Comment:
I was guessing that `image` was a typo, or does Docker refer to it's `versions` also as `images`?